doc/nrf/introduction.rst

```diff
@@ -55,10 +55,10 @@ The version numbers are incremented based on the following criteria:
 
 * The ``MAJOR`` version number is increased seldom, whenever a release is deemed to be introducing a large number of substantial changes across the board.
 * The ``MINOR`` version number is increased every time a major release is cut.
-  Major releases are the default types of an |NCS| release.
+  Minor releases are the default types of an |NCS| release.
   They introduce new functionality and may break APIs.
 * The ``PATCH`` version number is increased whenever a minor or bugfix release is cut.
-  Minor releases only address functional issues but do not introduce new functionality.
+  Patch releases only address functional issues but do not introduce new functionality.
 
```

Comment:
@carlescufi , @shantha-14 , would it make sense to include in this section, maybe around the line 70, a mention of the dev-tag releases as a "temporary freeze in time in the ``99`` version number that will later be replaced by a complete release"?

Reply:
@mbolivar-nordic  I think it would make sense to align and let customers know,

Reply:
"Replace" may be too strong of a word. I see the dev tags as being useful snapshots that mark where a significant new piece of functionality was introduced, but which may or may not have received the same amount of QA in other areas as a regular NCS release.

Reply:
also I am not sure if we should state 99 version number.  How will this be interpreted?  we will have dev tags which will not state 99.

Reply:
OK, so how about the following definition:
```
The SDK versions can sometimes include a dev tag, a snapshot of the development state that is created to highlight the introduction of a new piece of functionality. Unlike an NCS release, a dev tag may or may not have received the same amount of testing.
```
@shantha-14 , @bjorn-tore-taraldsen , @mbolivar-nordic ?

Reply:
@greg-fer that sounds good, but I would perhaps say "nRF Connect SDK versions can sometimes include a `-devN` postfix..."